src/Common/src/TypeSystem/Ecma/EcmaType.cs

```diff
@@ -42,6 +43,19 @@ internal EcmaType(EcmaModule module, TypeDefinitionHandle handle)
 #endif
         }
 
+        public override int GetHashCode()
+        {
```

Comment:
Should caching be extended to InstantiatedMethod.GetHashCode and MethodeDesc.GetHashCode?